addons/hr_holidays/tests/common.py

```diff
@@ -1,80 +1,62 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp.tests import common
+from odoo.tests import common
 
 
 class TestHrHolidaysBase(common.TransactionCase):
```

Comment:
1. why does this exist at all
2. why is it storing a ton of useless garbage (like env models) on the instance
3. why is it handling the case of missing system groups